docs/reference/contributing/target/rtc.md

```diff
@@ -22,8 +22,22 @@ The RTC HAL API provides a low-level interface to the Real Time Counter (RTC) of
 - Incorrect overflow handling.
 - Glitches due to ripple counter.
 
+#### Dependencies
+
+Hardware RTC capabilities.
+
 #### Implementing the RTC API
 
-RTC HAL API is in [hal/rtc_api.h](/docs/v5.7/mbed-os-api-doxy/rtc__api_8h_source.html). You need to implement the following functions to support RTC:
+You can find the API and specification for the RTC in the following header file:
+
+[![View code](https://www.mbed.com/embed/?type=library)](https://github.com/ARMmbed/mbed-os/blob/feature-hal-spec-rtc/hal/rtc_api.h)
```

Comment:
Please link to the Doxygen instead of to GitHub.

Reply:
The code is on a feature branch and the doxygen is only generated for master. In many cases that will be necessary as we need the porting guide to be up to date with what we want people to implement rather than master.